### ISSUE TITLE
Include support for tether npm module

### ISSUE THREAD

**Issue by essekia** [jrburke/notobo#4]
Include support for Tether npm module. A few react based elements use this. Or can you point me to how I  can do this myself?

**Comment by jrburke** [comment]
If the errors were of this form:

```
Could not convert: node_modules/tether/src/js/abutment.js, so skipping it. Error was: Error: Line 1: Unexpected token (
```

I just pushed a 0.0.6 release that updates esprima to handle the ES2015 syntax that is used those modules. Note that you likely need to convert those modules to ES5 syntax if you want them to work across all the major browsers, since they all do not have the same level of ES2015 support yet. Converting those modules is outside the scope of notobo.

So I think the core running of notobo for tether is fixed via the 0.0.6 release and the fix in issue #6.

**Comment by essekia** [comment]
Thats great! Thanks! 

Babel would help in the conversion so no problem.
